blogposts/2021/log4j-log4-shell-0-day.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Log4j Log4Shell 0-day: find, fix, and track affected code with code search"
+description: "How code search helps you deal with log4j CVE-2021-44228:
```

Comment:
Starting with "How code search helps..." feels self-serving. Can we either delete everything before the colon?

Reply:
@amieroth Agreed, I've suggested the change https://github.com/sourcegraph/about/pull/5019/files#r767804373